providers/base/units/graphics/test-plan.pxu

```diff
@@ -220,6 +218,8 @@ include:
     after-suspend-graphics/1_video_.*                        certification-status=blocker
     after-suspend-graphics/1_cycle_resolution_.*             certification-status=non-blocker
     suspend/1_xrandr_screens_after_suspend.tar.gz_auto
+    power-management/lid_close_suspend_open                  certification-status=blocker
+    power-management/lid                                     certification-status=blocker
```

Comment:
Any specific reason why the jobs order has been modified?

Reply:
The second close lid test will wait at least ${holdoff_timeout_sec} to sleep since the DUT just resume from last lid test.So it will take more time to finished this test than put it at other place which not just after a suspend. However, put this test to other place are not very appropriate too. And put it at here will test the hold off time in some way, so just put it back here.